src/main/java/org/glyptodon/guacamole/Stress.java

```diff
@@ -227,14 +247,24 @@ else if (instruction.getOpcode().equals("error")) {
 
                 // Otherwise, add total bytes
                 else {
-                    for (String arg : instruction.getArgs())
+                    for (String arg : instruction.getArgs()) {
+                        metrics.addBytes(arg.length());
                         bytes += arg.length();
+                    }
+
+                    // Add frames to metrics counter
+                    if (instruction.getOpcode().equals("png") ||
```

Comment:
The boundary of a frame is technically `sync`. Each frame can contain any number of `png` or `jpeg` instructions, not to mention other drawing operations. The current client implementation will actually only flush these drawing operations to the display surface upon receiving `sync`.

If we use only `png` and `jpeg` to count frames, the frame count will be inflated in the general case.